### DISCUSSION TITLE
Sqanti rescue: introducing duplicate transcripts in the gtf file

### DISCUSSION THREAD

**Discussion by ChrissiKalk97** [ConesaLab/SQANTI3#369] [Q&A] [answered]
Hi everyone,

I found that some transcripts appear twice in the gtf file I obtain after the Sqanti rescue step (I used a custom rules filter for the Sqanti filter step and ran the rescue step in the full mode):

`/scratch/fuchs/agschulz/kalk/Squanti3/final_assembly_rescue_06_12_24
grep 'ENST00000674426' rescue_custom_rules_filter_rescued.gtf 
1       PacBio  transcript      35883228        35930531        .       +       .       transcript_id "ENST00000674426"; gene_id "ENSG00000092847";
1       PacBio  exon    35883228        35883446        .       +       .       transcript_id "ENST00000674426"; gene_id "ENSG00000092847";
1       PacBio  exon    35888427        35888610        .       +       .       transcript_id "ENST00000674426"; gene_id "ENSG00000092847";
...

1       havana  transcript      35883209        35930532        .       +       .       transcript_id "ENST00000674426"; gene_id "ENSG00000092847"; gene_name "AGO1"
1       havana  exon    35883209        35883446        .       +       .       transcript_id "ENST00000674426"; gene_id "ENSG00000092847"; gene_name "AGO1";
1       havana  exon    35888427        35888610        .       +       .       transcript_id "ENST00000674426"; gene_id "ENSG00000092847"; gene_name "AGO1";
...
1       havana  CDS     35883422        35883446        .       +       0       transcript_id "ENST00000674426"; gene_id "ENSG00000092847"; gene_name "AGO1";
1       havana  CDS     35888427        35888610        .       +       2       transcript_id "ENST00000674426"; gene_id 
...`
For the gtf file of the QC and the filter step the same transcript ID was present but only as one transcript, so the duplication occurred somewhere in the rescue procedure.

If anyone knows why this happens or how to circumvent this, I would much appreciate you sharing your knowledge and experience :)

Best,

Christina

**Comment by carolinamonzo** [comment, marked as answer]
Hi, you should update the version of SQANTI you are using.
